src/services/validation.js

```diff
@@ -28,31 +28,22 @@
 *
 */
 
-// import validateFactory from '@natlibfi/marc-record-validate';
-// Import {} from '@natlibfi/marc-record-validators-melinda';
-
-export class ValidationError extends Error {
-	/* istanbul ignore next: Actual validation is currently not in use */
-	constructor(messages, ...params) {
-		super(params);
-		this.messages = messages;
-	}
-}
-
-export default async function () {
-//	Const validateFunc = await validateFactory([]);
-
-	return {validate};
-
-	async function validate(record) {
-		return [];
-		/* Const results = await validateFunc(record, {fix: true, validateFixes: true});
-
-		if (results.valid) {
-			return results.messages;
-		}
-
-		throw new ValidationError(results.messages);
-		*/
-	}
-}
+/* eslint-disable new-cap */
+import validateFactory from '@natlibfi/marc-record-validate';
+import {
+  FieldExclusion
+} from '@natlibfi/marc-record-validators-melinda';
+
+export default async () => {
+  const validate = validateFactory([await FieldExclusion([{tag: /^003$/u, value: /^(.(?<!FI-MELINDA))*?$/u}])]); // eslint-disable-line prefer-named-capture-group
```

Comment:
Validation should only validate, not fix. If the validation fails the record is considered invalid. See OpenAPI spec for how the error messages are passed to the client. I also updated the Jira task to be more specific about the validation.

Control fields validator is propably the right one for this use case.